ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/transaction/UtxoLedgerTransaction.java

```diff
@@ -288,4 +289,7 @@ default List<ContractState> getOutputContractStates() {
     // TODO : Rename to getOutputContractStates
```

Comment:
Can you remove this `TODO`s in this class while you're here since they'll never be changed now (above `getOutputStates` and `getInputStates`).